.prettierignore

```diff
@@ -9,3 +9,5 @@ out
 
 # We don't maintain these files
 src/api/status/public/assets
+
+src/web/public
```

Comment:
This seems like something that should happen on `master` in a separate PR vs. here (this is an unrelated change).  If w need this, can you do a separate PR just for this change?

Reply:
Yes, I'll remove it from this PR. I originally added this because two files were automatically created in `src/web/public`: `sw.js` and `workbox-0ba03b8b.js` (git ignores both).
I don't know how that happened. I discovered them running `pnpm prettier-check`